osu.Game/OsuGame.cs

```diff
@@ -1136,12 +1136,19 @@ private void forwardTabletLogsToNotifications()
 
                 if (entry.Level == LogLevel.Error)
                 {
-                    Schedule(() => Notifications.Post(new SimpleNotification
+                    Schedule(() =>
                     {
-                        Text = $"Encountered tablet error: \"{message}\"",
-                        Icon = FontAwesome.Solid.PenSquare,
-                        IconColour = Colours.RedDark,
-                    }));
+                        Notifications.Post(new SimpleNotification
+                        {
+                            Text = $"Disabling tablet support due to error: \"{message}\"",
+                            Icon = FontAwesome.Solid.PenSquare,
+                            IconColour = Colours.RedDark,
+                        });
+
+                        var tabletHandler = Host.AvailableInputHandlers.OfType<ITabletHandler>().FirstOrDefault();
+                        if (tabletHandler != null)
+                            tabletHandler.Enabled.Value = false;
```

Comment:
I really dislike seeing code like this because yes, right now we only have only one `ITabletHandler` implementation, but if we ever *somehow* happen to have more, then this can start failing in weird and unpredictable ways because it's not going to be disabling the *correct* tablet handler that is spewing errors.

Can we either make this a `SingleOrDefault()` to at least notify us that the invariant no longer holds, or a `foreach` loop that, yes, by all means currently will iterate over only one item, but if another ever appears, then it will be handled automagically?

Reply:
I did consider this. I think that if another handler gets added there's 99% chance this would be found in review, because for instance the settings implementation will have weird issues (double settings sections with no visual differentiation, at very least) which would trigger checking all `ITabletHandler` usage.

Switching to `SingleOrDefault` would introduce a potential user facing crash, so will loop over all for now I guess.